src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/service/OIDCConfigurationService.java

```diff
@@ -59,8 +59,7 @@ private void loadConfig() {
 			uri = new URI(ecrcProps.getOauthWellKnown());
 			config = restTemplate.getForObject(uri, OIDCConfiguration.class);
 		} catch (URISyntaxException e2) {
-			logger.error("Unable to load remote server Well-Known configuration endpoint. Check Oauth2 well-known endpoint configuration. " + e2.getMessage(), e2); 
-			e2.printStackTrace();
+			logger.error("Unable to load remote server Well-Known configuration endpoint. Check Oauth2 well-known endpoint configuration. ", e2);
```

Comment:
I know this isn't directly because of your code change, but should the capitalization of "well-known" be more consistent here?

Reply:
Good call I have updated this as well.